slmon/apps/slmon/slmon.py

```diff
@@ -128,21 +128,22 @@ def write(self, f):
         lines.sort()
         f.write('\n'.join(lines)+'\n')
 
-
 def colorLegend(htmlfile):
     htmlfile.write("<p><center>Latencies:<br>\n" \
```

Comment:
Since this code is touched you may further increase the readability by using multi-line strings, e.g.,
```python
    print("""<p>
  <center>Latencies:<br>
    <table cellpadding="2" cellspacing="1" border="0" bgcolor="#000000">
      <tr>
        <td bgcolor='#FFFFFF'><b>&lt; 1 min</b></td>
        ...
      </tr>
    </table>
  </center>
</p>
""", file=htmlfile)
    

```